app/src/main/java/com/acsbendi/requestinspectorwebview/RequestInspectorJavaScriptInterface.kt

```diff
@@ -199,16 +216,26 @@ internal class RequestInspectorJavaScriptInterface(webView: WebView) {
             val formParameter = formParameterJsonArray.get(i) as JSONObject
             val name = formParameter.getString("name")
             val value = formParameter.getString("value")
-            if (i != 0) {
-                resultStringBuilder.append("\n")
+            val checked = formParameter.optBoolean("checked")
+            val type = formParameter.getString("type")
+
+            if (!abandon(type, checked)) {
+                if (i != 0) {
+                    resultStringBuilder.append("\n")
+                }
+                resultStringBuilder.append(name)
+                resultStringBuilder.append("=")
+                resultStringBuilder.append(value)
             }
-            resultStringBuilder.append(name)
-            resultStringBuilder.append("=")
-            resultStringBuilder.append(value)
+
         }
         return resultStringBuilder.toString()
     }
 
+    private fun abandon(type: String, checked: Boolean): Boolean {
```

Comment:
Could we find a better name for documentation? How about `isExcludedFormParameter` or `IsFormParameterExcludedFromRequest`?

Reply:
done